hook/newrelic.go

```diff
@@ -0,0 +1,72 @@
+package hook
+
+import (
+	"bytes"
+	"errors"
+	"github.com/lucasgomide/snitch/types"
+	"net/http"
+	"time"
+)
+
+type NewRelic struct {
+	Host             string
+	ApplicationId    string
+	ApiKey           string
+	Revision         string
+}
+
+func (s NewRelic) CallHook(deploys []types.Deploy) error {
+	httpClient := &http.Client{
+		Timeout: time.Second * 10,
+	}
+	if err := s.createDeploy(httpClient, deploys[0]); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (s *NewRelic) createDeploy(httpClient *http.Client, deploy types.Deploy) error {
+	data := []byte(`
+	{ "deployment": {
```

Comment:
Can you improve data's formatting, for better legibility ?

Reply:
Sure, fixed!